v23.1/execution-locality-backups.md

```diff
@@ -0,0 +1,100 @@
+{{site.data.alerts.end}}
+
+## Syntax
+
+To specify the locality requirements for the coordinating node, run `EXECUTION LOCALITY` with key-value pairs. The key-value pairs represent the [locality designations](cockroach-start.html#locality) assigned to the cluster at startup.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BACKUP DATABASE {database} INTO 'external://backup_storage' WITH EXECUTION LOCALITY = 'region={region},cloud={cloud}';
+~~~
+
+When you run `EXECUTION LOCALITY`, consider the following:
+
+- The backup job will fail if no nodes match the locality filter.
+- Selection of the coordinating node that matches the locality filter may slightly increase the startup latency of the backup job.
+- Even though a backup job has been pinned to a locality, it may still read data from another locality if there are no replicas in its locality.
+- If the job is created on a node that does not match the locality filter, you will receive an error even when the **job creation was successful**. This error indicates that the job execution moved to another node. To avoid viewing this error, we recommend using the [`DETACHED`](backup.html#detached) option with `EXECUTION LOCALITY` when creating a manual job rather than a [scheduled job](create-schedule-for-backup.html). Then, use the [`SHOW JOB WHEN COMPLETE`](show-jobs.html#show-job-when-complete) statement to determine when the job has finished. See [Job coordination using the execution locality option](#job-coordination-using-the-execution-locality-option) for more detail.
+
+## Job coordination using the execution locality option
+
+When you start or [resume](resume-job.html) a backup with `EXECUTION LOCALITY`, it is necessary to determine the [coordinating node for the job](backup-architecture.html#job-creation-phase). If a node that does not match the locality filter is the first node to claim the job, it will find a node that does match the filter and transfer the execution to it. This can result in a short delay in starting or resuming a backup job that has execution locality requirements.
+
+If you create a backup job on a [gateway node](architecture/sql-layer.html#overview) with a locality filter that does **not** meet the filter requirement in `EXECUTION LOCALITY`, the job will return an error indicating that it moved execution to another node. This error is returned because when you create a job without the [`DETACHED`](backup.html#detached) option, the [job execution](backup-architecture.html#resolution-phase) needs to be run to completion by the gateway node while it is still attached to the SQL client to return the result. 
+
+Once the coordinating node is determined, it will [assign chunks of row data](backup-architecture.html#resolution-phase) to nodes that are then responsible for reading that row data and backing it up. The coordinator will only assign row data to nodes that match the locality filter. The following will happen in different cases:
```

Comment:
```suggestion
Once the coordinating node is determined, it will [assign chunks of row data](backup-architecture.html#resolution-phase) to eligible nodes, and each node reads its assigned row data and backs it up. The coordinator will assign row data only to those nodes that match the backup job's locality filter. The following situations could occur:
```

Reply:
Yes! This is much more clear